frontend/craco.config.js

```diff
@@ -1,12 +1,48 @@
 const path = require('path');
+const fs = require('fs');
+
+const WorkboxWebpackPlugin = require('workbox-webpack-plugin');
+const paths = require('./paths'); // <-- Copied from "CRA eject" -- "config" folder.
+
+const isEnvProduction = process.env.NODE_ENV === "production"
+const swSrc = paths.swSrc;
+console.info({swSrc, exists: fs.existsSync(swSrc)})
 
 module.exports = {
-  // Output to ./course (instead of ./build)
   webpack: {
     configure: (webpackConfig, {env, paths}) => {
+      // Output to ./course (instead of ./build)
       paths.appBuild = webpackConfig.output.path = path.resolve('course');
       return webpackConfig;
     },
+    plugins: [
```

Comment:
Doing it at `plugins` rather than `webpack.configure` did not work.

<details><summary>Original error (no longer relevant)</summary>

[0be6cb3](https://github.com/alexmojaki/futurecoder/commit/0be6cb3970a07cf2e7830872a51bf32a477597d5) results in this error:

```
> frontend@0.1.0 build <redacted>/futurecoder/frontend
> craco build

{
  swSrc: '<redacted>/futurecoder/frontend/src/service-worker.js',
  exists: true
}
Creating an optimized production build...
Failed to compile.

Can't find self.__WB_MANIFEST in your SW source.
```

... which is unexpected because `src/service-worker.js` definitely has `self.__WB_MANIFEST` in it, and it works fine when we don't override the config.

</details>